conf.d/network.yaml.default

```diff
@@ -13,3 +13,6 @@ instances:
     # Optionally completely ignore any network interface
     # matching the given regex:
     # excluded_interface_re: my-network-interface.*
+
+    # Do not combine connection states
```

Comment:
Could you add more details on what the option does? And I think it's worth explicitly mentioning what the default value is.